### ISSUE TITLE
Mehrfachauswahl im Hauptfenster hinzufügen

### ISSUE THREAD

**Issue by flori2212** [flori2212/EnergieverbrauchDerSchule#1]
Im Hauptfenster (mainwindow.xaml) soll man mehrere "Consumer" auswählen können, und diese dann auch gleichzeitig bearbeiten.
In den ComboBoxen soll während einer Mehrfachauswahl stehen "mehrere Elemenete ausgewählt" und wenn man ein Element in der ComboBox ausgewählt wird, soll dies auf alle Items angewendet werden

**Comment by NoFear23m** [comment]
Jo, ich kann dir dabei helfen, dafür muss aber das komplette ViewModel umgebaut werden.
Ich versuche es mal zu erklären

1.) Willst du erkennen das mehrere Consumer markiert sind brauchst du im ConsumerViewModel ein Property IsSelected. 
Dann einen Style für die DataGridRow:
` <Style TargetType="DataGridRow">
            <Setter Property="IsSelected"
                    Value="{Binding IsSelected}" />
        </Style>`

Aber das ist schon ein Thema das andere bestimmt auch interessiert, das wäre im Forum sicher toll aufzuzeigen ;-)

Grüße

**Comment by flori2212** [comment]
Ich habe es auch ertmal auf den Milestone "Letze Änderung" gesetzt, weil ich das ganz am Ende machen würde.
Dann würde ich auch einen passenden Forumbeitr schreiebn

**Comment by NoFear23m** [comment]
Ah, ok. Habe ich übersehen.
